app/sidekiq/form526_submission_failure_email_job.rb

```diff
@@ -0,0 +1,115 @@
+# frozen_string_literal: true
+
+require 'va_notify/service'
+
+class Form526SubmissionFailureEmailJob
+  include Sidekiq::Job
+
+  attr_reader :submission_id
+
+  STATSD_PREFIX = 'api.form_526.veteran_notifications.form526_submission_failure_email'
+  # https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/274bea7fb835e51626259ac16b32c33ab0b2088a/platform/practices/zero-silent-failures/logging-silent-failures.md#capture-silent-failures-state
+  DD_ZSF_TAGS = [
+    'service:disability-application',
+    'function:526_backup_submission_to_lighthouse'
+  ].freeze
+
+  sidekiq_options retry: 14
+
+  sidekiq_retries_exhausted do |msg, _ex|
+    job_id = msg['jid']
+    error_class = msg['error_class']
+    error_message = msg['error_message']
+    form526_submission_id = msg['args'].first
+    timestamp = Time.now.utc
+
+    Rails.logger.warn(
+      'Form526SubmissionFailureEmailJob retries exhausted',
+      {
+        job_id:,
+        timestamp:,
+        form526_submission_id:,
+        error_class:,
+        error_message:
+      }
+    )
+
+    StatsD.increment("#{STATSD_PREFIX}.exhausted")
+    StatsD.increment('silent_failure', tags: DD_ZSF_TAGS)
+  rescue => e
+    Rails.logger.error(
+      'Failure in Form526SubmissionFailureEmailJob#sidekiq_retries_exhausted',
+      {
+        job_id:,
+        messaged_content: e.message,
+        submission_id:,
+        pre_exhaustion_failure: {
+          error_class:,
+          error_message:
+        }
+      }
+    )
+    raise e
+  end
+
+  def perform(submission_id)
+    submission = Form526Submission.find(submission_id)
+    send_email(submission)
+    track_remedial_action(submission)
+    log_success(submission)
+  rescue => e
+    log_failure(e, submission)
+    raise
+  end
+
+  private
+
+  def send_email(submission)
+    email_client = VaNotify::Service.new(Settings.vanotify.services.benefits_disability.api_key)
+    template_id = Settings.vanotify.services.benefits_disability.template_id
+                          .form526_submission_failure_notification_template_id
+
+    email_client.send_email(
+      email_address: submission.veteran_email_address,
+      template_id:,
+      personalisation: {
+        first_name: submission.get_first_name,
+        date_submitted: submission.format_creation_time_for_mailers
+      }
+    )
```

Comment:
I think there should be a surrounding rescue block around this, or somewhere, that rescues and increments silent failure before raising? Like if the code to send an email errors, and we still havent sent the email, we still need to imcrement the silent failure counter... unless thats happening somewhere and Im not seeing it?

Reply:
so i don't think we want to increment on failure, but rather exhaustion.  Otherwise we will be counting 14 silent failures as this retries

Reply:
You are right, I put this comment in the wrong place. Marking resolved.